opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java

```diff
@@ -383,4 +385,30 @@ protected void evaluateExclusiveBreastFeeding(Map<String, ServiceWrapper> servic
                 .build();
         actionList.put(title, action);
     }
+
+    private void evaluateChildSafety(Map<String, ServiceWrapper> serviceWrapperMap) throws Exception {
+        ServiceWrapper serviceWrapper = serviceWrapperMap.get("Child Safety");
```

Comment:
The referenced recurring service starts with a lowercase letter, resulting in a null serviceWrapper object